### ISSUE TITLE
ALAC 

### ISSUE THREAD

**Issue by legosoff** [staxrip/staxrip#1170]
Hi, Dendraspis! Why is there no ALAC format, do I need to add ALAC when it will be updated?

**Comment by Dendraspis** [comment]
Good question,  but why should we need or use it? `FLAC` is besser supported and more common.  🤔
Is there a reason why anyone would use `ALAC` over `FLAC` ?

**Comment by legosoff** [comment]
Yes FLAC webOS up to 2 channels to play, 5.1 and 7.1 channels not read. ALAC webOS 5.1 and 7.1 channels are working. The HLS browser, soundbar.

**Comment by legosoff** [comment]
Please update StaxRip 2.31.0 and don't forget ALAC qaac. 
Thank you.

**Comment by legosoff** [comment]
Hi,  Dendraspis. StaxRip v2.34.0 that's how it works great Dolby Vision metadata CROP. I asked you, why don't you do it ALAC? When will it be?

**Comment by Dendraspis** [comment]
Other bugs and features have higher priority. 

And according to this, `ALAC` is not supported at all:
https://webostv.developer.lge.com/develop/specifications/video-audio-230#audio-coding-format

**Comment by legosoff** [comment]
It's weird, I have an ALAC playing LG 1C running. Okay, there's no need for that anymore. I am using ATMOS 7.1 1664 bitrate JOC.  Thanks for RPU crop.

**Comment by legosoff** [comment]
Please delete everything here aac ALAC #1170

**Comment by backryun** [comment]
Dendraspis, Please cut off my before account and transfer I dm'ed you.
